rclpy/rclpy/node.py

```diff
@@ -761,6 +763,33 @@ def _set_parameters_atomically(
 
         return result
 
+    def add_on_set_parameters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """
+        Add a callback in front to the list of callback.
```

Comment:
@ivanpauno Can you comment why the C++ implementation inserts new callbacks *before* the already registered ones?